CMakeLists.txt

```diff
@@ -1,4 +1,4 @@
-cmake_minimum_required(VERSION 3.12)
+cmake_minimum_required(VERSION 3.21)
```

Comment:
what requires it ?

Reply:
Well, after a second though, it might not be required. We need it to compile f3d but I guess it's not the SB responsibility to define the minimum required version for f3d?

Reply:
not at all, this is about the superbuild cmake code